src/components/common/NavBar.js

```diff
@@ -3,13 +3,14 @@ import { Link } from 'react-router-dom';
 
 /**
  * Required Props:
+ * brandTitle {string} - String representing the website brand name
+ * toggleSideBar {function} - Callback for toggling the sidebar
+ * SubModuleTitle {String} - Title of current submodule
  * navBarType {String} - String representing which version of the navbar to render
  *    Nav Bar Types: 'module', 'homepage', 'catalog'
- *
- * TODO: Include props for other nav bar information/links
  */
 function NavBar(props) {
-  const { navBarType } = props;
+  const { brandTitle, toggleSideBar, SubModuleTitle, navBarType } = props;
```

Comment:
I'm wondering if we do actually need the brandTitle to be a prop since the name of our website is not bound to change that often, plus it means we unnecessarily have to redefine the website name as prop every time we include the nav bar on a web page.

Reply:
No I don't think we do. Idk why I left that, will remove it.